ui/src/shared/components/LayoutCell.tsx

```diff
@@ -18,6 +18,11 @@ import {NoteVisibility} from 'src/types/dashboards'
 import {TimeSeriesServerResponse} from 'src/types/series'
 import {CellType} from 'src/types/dashboards'
 
+enum VisType {
+  Graph,
+  Table,
```

Comment:
purely an aesthetic debate, but I wonder if this should be called "Raw"?

Reply:
I personally think Graph/Table is more clear as far as what the user is going to see